olm/quickstarts/rhosak-openshift-connect-quickstart.yaml

```diff
@@ -3,160 +3,328 @@ kind: ConsoleQuickStart
+        you'll log in to the RHOAS CLI using a token.
+
+        #### Prerequisites
+         - You've created an application in your project that runs the required CLI tools.
+         - The RHOAS Operator is installed on your OpenShift cluster.
+
+        #### Procedure
+
+        1. In the OpenShift web console, click the [perspective switcher]{{highlight qs-perspective-switcher}}. Switch to the **Developer** perspective.
+
+           The **Topology** page opens.
+
+        1. Ensure that the current OpenShift project is the one you created earlier in this quick start.
+
+            i. At the top of the **Topology** page, click the **Project** drop-down menu.
+
+            ii. Select the project that you previously created.
 
-        The Red Hat OpenShift Application Services operator allows you to represent Red Hat Cloud Services as first class citizens in your OpenShift environment.
-        This enables you to work and integrate with these services using standard OpenShift/Kubernetes features and APIs.
+        1. If you don't already have a command-line window open in the pod for your tools application, open one, as described earlier in this quick start.
 
-        We will use the `rhoas` CLI to connect your Kafka instance to your OpenShift project. The `rhoas` CLI uses a browser-based login flow by default
-        (meaning, when you execute `rhoas login` it will start a sign-in flow in your web browser). Since our terminal in OpenShift does not have access to a browser,
-        we need to login using a token.
+        1. Log in to the OpenShift CLI using a token.
 
-        1. Navigate to [https://cloud.redhat.com/openshift/token](https://cloud.redhat.com/openshift/token) and copy the token to your clipboard using the **Copy to clipboard** button.
+            i. In the upper-right corner of the console, next to your user name, click the drop-down menu. Select **Copy login command**.
+               A new page opens.
 
-        1. In your terminal, execute the RHOAS CLI login command, replacing *{token}* with the token you've just copied to your clipboard: `rhoas login --token={token}`.
-        You should see a response saying: **You are now logged in as {user}**.
+            ii. Click the **Display Token** link.
 
-        1. List your Kafka instances by executing the command: `rhoas kafka list`. You should see a Streams for Apache Kafka instance that has been provisioned for you.
+            iii. In the section entitled **Log in with this token**, copy the full `oc login` command shown.
 
-        The **rhoas** CLI uses the **oc** CLI to determine the OpenShift instance, and project, that the Streams for Apache Kafka instance should be connected to.
-        Let's verify that your **oc** client is properly connected to your OpenShift project.
+            v. On the command line, paste the login command you copied. Right-click on the command line and select **Paste**. An example is shown below.
```

Comment:
I believe IBM is against "above" and "below" etc., so suggest something like, "The following is an example login command:"  However, although that's the route I tend to go, IBM is also against unmodified instances of "following", so IBM would prefer something like "The following code is an example login command." I just tend to flout that myself to streamline. Up to you.